app/views/jst/conferences/newConference.handlebars

```diff
@@ -88,24 +94,32 @@
   <div class="ig-sublist" id="conference-{{id}}" style="display: none;">
     <ul>
       {{#each recordings}}
-        <li>
-          <div class="ig-row">
+        <li class="recording">
+          <div class="ig-row" data-id="{{recording_id}}" style="text-align: center;">
             <div class="ig-row__layout">
               <div class="ig-info">
                 <div class="ig-details">
-                  <a href="{{playback_url}}" target="_blank" class="ig-title">
+                  <a href="{{playback_url}}" target="_blank" class="ig-title"
+                    style="line-height: inherit;" data-id="{{recording_id}}" data-format="presentation">
                     {{title}}
                   </a>
                   {{dateString created_at}}
                   &nbsp;&#124;&nbsp;
-                  {{minutesToHM duration_minutes}}
+                  {{durationToString duration_minutes}}
                 </div>
               </div>
-              <div class="ig-button">
-                <a class="btn btn-small" target="_blank" href="{{playback_url}}">
-                  {{#t "view"}}View{{/t}}
-                </a>
-              </div>
+              {{#if ../permissions.delete}}
+                <div class="ig-loader" data-id="{{recording_id}}" data-action="delete" style="display: none;">
+                  <img src="/images/ajax-loader-black-on-white.gif" class="loader" />
+                </div>
+                <div class="ig-button" data-id="{{recording_id}}" data-action="delete" data-url="{{../../url}}">
+                  <a href="#"
+                      class="btn btn-small icon-trash delete_recording_link"
+                      aria-label="{{#t "delete"}}Delete{{/t}}"
```

Comment:
We shouldn't need this label.  Let the link describe what it actually is.  The label will make it more confusing.